docs/hydra/guides/deploy-hydra-example.mdx

```diff
@@ -0,0 +1,494 @@
+---
+id: deploy-hydra-example
+title: Deploy to production
+---
+
+import { useLatestRelease } from '@site/src/hooks'
+import CodeBlock from '@theme/CodeBlock'
+
+This guide explains how to setup and run Ory Hydra in an exemplary production environment. It uses Postgres as database, Nginx as
+reverse proxy, and Digital Ocean as cloud provider. You can use another
+[relational database](https://www.ory.sh/docs/ecosystem/deployment#data-storage-and-persistence), a different reverse proxy,
+deploy on any other cloud host, and
+[spin up a custom user interface in your favorite language](https://www.ory.sh/docs/hydra/sdk/) - this is just an example!
+
+The guide covers the following topics:
+
+- Installation and configuration of [Postgres](https://www.postgresql.org/)
+- Installation and configuration of [Ory Hydra](../install.mdx)
+- Installation and configuration of [Nginx](https://www.nginx.com/)
+
+## Create a Droplet
+
+Spin up a droplet with the following configuration:
+
+- **OS**: Ubuntu 20.04
+- **Plan**: Basic
+- **CPU options**: Regular with SSD
+- **RAM**: 1Gb
+- **SSD**: 25Gb
+- **VPC network**: default
+- **Authentication**: SSH Keys. Don't forget to add your own SSH key
+- **Region**: Choose your region
+
+:::note
+
+This example shows a single configuration on a small droplet. The configuration of a virtual machine (VM) may vary depending on
+the scale of your application.
+
+:::
+
+Wait for the virtual machine (VM) to start up and copy the IP address.
+
+This guide uses `oauth2.example.com` as a hostname to run Ory Hydra. Replace it with `<something>.<your-domain>` for your
+purposes. You need to configure DNS, create an `A type` record, and point it to the VM's IP.
+
+## Install required dependencies
```

Comment:
It'd make sense to turn this paragraph into an ordered list to show the flow better.